scripts/components/LviCartItem.ts

```diff
@@ -49,10 +51,34 @@ export default class LviCartItem extends LviCartItemDesign {
         });
     }
     get productPrice(): string {
-        return this.lblProductPrice.text;
+        return this.tvPrice.text;
     }
     set productPrice(value: string) {
-        this.lblProductPrice.text = value;
+        const discountExists = !!this.tvDiscount.text;
+        let attributeString = new AttributedString({
+            strikethrough: discountExists,
+            ios: {
+                strikethroughColor: themeService.getNativeStyle('.product-price').textColor
+            },
+            string: value || '',
+            font: themeService.getNativeStyle(discountExists ? '.product-price.discount' : '.product-price.nodiscount').font,
+            foregroundColor: themeService.getNativeStyle(discountExists ? '.product-price.discount' : '.product-price.nodiscount').textColor
+        });
+        this.tvPrice.scrollEnabled = false;
+        this.tvPrice.attributedText = [attributeString];
+    }
+    get productDiscount(): string {
+        return this.tvDiscount.text;
+    }
+    set productDiscount(value: string) {
+        let attributeString = new AttributedString({
+            string: value || '',
+            font: themeService.getNativeStyle('.product-price.nodiscount').font,
```

Comment:
Getting theme values should top on the page class.

Reply:
DONE.